builder/src/bin/permissioned-builder.rs

```diff
@@ -133,24 +133,43 @@ pub struct PermissionedBuilderOptions {
     pub state_peers: Vec<Url>,
 
     /// Port to run the builder server on.
-    #[clap(short, long, env = "BUILDER_SERVER_PORT")]
+    #[clap(short, long, env = "ESPRESSO_BUILDER_SERVER_PORT")]
     pub port: u16,
 
     /// Port to run the builder server on.
-    #[clap(short, long, env = "BUILDER_ADDRESS")]
+    #[clap(short, long, env = "ESPRESSO_BUILDER_ADDRESS")]
```

Comment:
Is this variable used? The address is obtained from the private key derived from the mnemonic.

Reply:
Actually nevermind, I thought we will remote the permissioned builder anyway, so no need to fix the code I think.